### PR TITLE
Statement Distribution Guide Edits

### PR DESCRIPTION
**Pull request by BradleyOlson64**
Nits and clarifications for the statement distribution guide. Also cleans up some unneeded toml and comment lines.

### CODE REVIEW

File: roadmap/implementers-guide/src/node/backing/statement-distribution.md [hunk 1]
```diff
@@ -171,19 +171,19 @@ backing subsystem itself.
...
+- `BackedCandidateManifest`
+  - Notification of a backed candidate being known by the sending node.
+  - For the candidate being requested by the receiving node if needed.
+  - Announcement
+  - Handled by `handle_incoming_manifest`
+- `BackedCandidateKnown`
+  - Notification of a backed candidate being known by the sending node.
+  - For informing a receiving node which already has the candidate.
+  - Acknowledgement.
+  - Handled by `handle_incoming_acknowledgement`
```
Comment (mrcnski) [inline roadmap/implementers-guide/src/node/backing/statement-distribution.md:186]: These are the possible messages for `NetworkBridgeUpdate`, which is why they were indented like that. I see it's a bit confusing. Maybe we can keep your indentation and just add a new section, like this:

```md
- `StatementDistributionMessage::NetworkBridgeUpdate`
  - Handled by `handle_network_update`
  - Possible events: see next section.

#### Network bridge events

- v1 compatibility
- `Statement`
  - Notification of a signed statement.
[...]
```

File: roadmap/implementers-guide/src/node/backing/statement-distribution.md [hunk 2]
```diff
@@ -443,10 +448,16 @@ description of the flow. See module-docs for full details.
...
     chain block yet.
-- **Fragment tree:** A parachain fragment not referenced by the relay-chain.
-  It is a tree of prospective parachain blocks.
+- **Fragment:** A prospective para block (that is, a block not yet referenced by
+  the relay-chain). Fragments are anchored to the relay-chain at a particular
+  relay-parent.
+- **Fragment tree:**
+  - A tree of fragments. Together, these fragments define one or more
+    prospective paths a parachain's state may transition through.
+  - See the "Fragment Tree" section.
```
Comment (mrcnski) [inline roadmap/implementers-guide/src/node/backing/statement-distribution.md:457]: Should be:

```suggestion
  - See the "Fragment Tree" section on the Prospective Parachains page.
```

ideally with a hyperlink to that page's section. But, [this PR](https://github.com/paritytech/polkadot/pull/6933) hasn't been merged yet, so this change should probably go in that PR.

Comment (mrcnski) [reply roadmap/implementers-guide/src/node/backing/statement-distribution.md:457]: Or, we can have a shared glossary for the backing subsystems, i.e. a separate Glossary subpage. I personally think it's nice to have a glossary on the same page so readers don't have to hunt for it, and to have terms only relevant to what they're reading about, but it's an idea.

**Review by mrcnski — CHANGES_REQUESTED** [review]
Nice clarifications. 👍 Just a couple changes needed.

File: roadmap/implementers-guide/src/node/backing/statement-distribution.md (hunk 2 above)
Comment (BradleyOlson64) [reply roadmap/implementers-guide/src/node/backing/statement-distribution.md:457]: Makes sense that it should go in the prospective parachains PR 👍

Comment (BradleyOlson64) [reply roadmap/implementers-guide/src/node/backing/statement-distribution.md:457]: Thanks for the review!

**Review by slumber — APPROVED** [review]

**Pull request merged**